### PR TITLE
Another CLI change fix

### PR DESCRIPTION
**Pull request by ocefpaf**
I guess there will be more failures after this one.  Don't merge it yet.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Not sure how to proceed with the feedstocks that failed.  I deleted the empty `automake`, `autoconf`, and `m4`. (Will re-submit as soon as this issue is fixed.)

The `pyinotify-feedstock` also seems empty.  I let's see if that is fixed with this PR.

File: .CI/create_feedstocks.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def write_token(name, token):
         # Break the previous loop to allow the TravisCI registering to take place only once per function call.
         # Without this, intermittent failiures to synch the TravisCI repos ensue.
         for feedstock_dir, name, recipe_dir in feedstock_dirs:
-            subprocess.check_call(['conda', 'smithy', 'register-ci', feedstock_dir] + owner_info)
+            subprocess.check_call(['conda', 'smithy', 'register-ci', '--feedstock_directory', feedstock_dir] + owner_info)
```
Comment (JanSchulz) [inline .CI/create_feedstocks.py]: Maybe use `subprocess.check_output(...)` https://docs.python.org/2/library/subprocess.html#subprocess.check_output and then wrap the whole thing in a 

``` 
try:
   ... all the subprocess calls
except subprocess.CalledProcessError as e:
    print("Failed with returncode: %s\nOutput:\n%s\n" %(e.returncode, e.output))
    sys.exit(-1)
```

Sorry for all the breakage: when I did the changes, I didn't even know the `.CI` subdirectory existed... :-/

Comment (ocefpaf) [inline .CI/create_feedstocks.py]: You have a point there.  Maybe the `.CI` should not be hidden.

**Comment by pelson** [comment]
> Not sure how to proceed with the feedstocks that failed. I deleted the empty automake, autoconf, and m4. (Will re-submit as soon as this issue is fixed.)

The scripts will continue where they left off if you have an empty feedstock. Should have been solved by just merging this. Thanks @ocefpaf

**Comment by ocefpaf** [comment]
> Should have been solved by just merging this. Thanks @ocefpaf

I ended up removing `autoconf`, `automake`, and `m4` empry feedstocks. (That happened before I sent this PR.) The rest was OK.

PS: I am not sure it is worth re-submitted those packages as the new docker image will have those.

**Comment by pelson** [comment]
Looks like there aren't any dangling feedstock repos, so essentially we are at the point where it is as if those recipes hadn't been submitted. I'm happy either way - I guess eventually we may want recipes for these things, but until the day comes, let's not make work for ourselves :wink: 

Thanks @ocefpaf

**Pull request merged**